lib/bundler/cli/outdated.rb

```diff
@@ -82,5 +94,12 @@ def run
         exit 1
       end
     end
+
+  private
+
+    def minor_trim(spec)
+      segments = spec.version.segments
```

Comment:
I'd tested with one segment and after segments method it returned an array with only one element(no nil element), but I'll add the compact just to be sure.